jargon-core/src/test/java/org/irods/jargon/core/pub/CollectionAndDataObjectListAndSearchAOImplTest.java

```
@@ -1993,19 +1993,19 @@ public void testListDataObjectsUnderPathDoesNotContainDuplicatesAtPageBoundary()
 		//
 
 		// Remove all data objects.
-		for (int i = 0; i < dataObjectCount; i++) {
-			irodsFile = irodsFileSystem.getIRODSFileFactory(irodsAccount)
-					.instanceIRODSFile(rootCollection + "/data_object." + i);
-			irodsFile.delete();
-			irodsFile.close();
-		}
+//		for (int i = 0; i < dataObjectCount; i++) {
+//			irodsFile = irodsFileSystem.getIRODSFileFactory(irodsAccount)
+//					.instanceIRODSFile(rootCollection + "/data_object." + i);
+//			irodsFile.delete();
+//			irodsFile.close();
+//		}
 
 		// Remove temporary resources.
-		rao.removeChildFromResource(replResc.getName(), ufs0Resc.getName());
-		rao.removeChildFromResource(replResc.getName(), ufs1Resc.getName());
-		rao.deleteResource(ufs0Resc.getName());
-		rao.deleteResource(ufs1Resc.getName());
-		rao.deleteResource(replResc.getName());
+//		rao.removeChildFromResource(replResc.getName(), ufs0Resc.getName());
+//		rao.removeChildFromResource(replResc.getName(), ufs1Resc.getName());
+//		rao.deleteResource(ufs0Resc.getName());
+//		rao.deleteResource(ufs1Resc.getName());
+//		rao.deleteResource(replResc.getName());
```

Why would we not remove the resources created by this test?

---

Those are commented out because I failed to remove the replicas _(forgot to include the force option on the unlink calls)_.

Here are a few reasons why I'm thinking of not cleaning up.
- If a test fails during a full run, rerunning the tests using the same containers does not produce reliable results
  - I always relaunch the containers to make sure the environment is pristine 
- It's faster to simply skip cleaning up the 5k files
- There's no chance for the test to fail during clean up
  - This is obviously something that can be worked around - i.e. ignore exceptions/errors, but that puts us right back to where we started
- The resources created by the test cannot break other tests

Thoughts?

---

i think we always want all tests to clean up?    anything else is cognitive load on the tester i think?

---

Ok. Will see if I can get clean up working.